tox.ini

```diff
@@ -33,11 +31,16 @@ setenv =
     py39: INTERP=py39
     pypy3: INTERP=pypy3
     nocov: PYTHONASYNCIODEBUG=1
-    !py39: PY_39=lt
+    py36: PY_36=eq
+    !py36: PY_36=gt
     py39: PY_39=ge
+    !py39: PY_39=lt
     PLATFORM={env:PLATFORM:linux}
 passenv =
     PYTHON*
+    TRAVIS
+    CI
+    GITHUB*
```

Comment:
`TRAVIS` (if set) and `CI` required by `housekeep.py`.

`GITHUB*` added for future needs.